parm/post_avblflds.xml

```diff
@@ -6432,23 +6512,23 @@
     <param>
        <post_avblfldidx>702</post_avblfldidx>
        <shortname>GSD_MAX_LTG_THREAT1_ON_ENTIRE_ATMOS</shortname>
-       <pname>LTGTHREAT1</pname>
+       <pname>LTNGTH1</pname>
        <fixed_sfc1_type>entire_atmos</fixed_sfc1_type>
        <scale>3.0</scale>
     </param>
 
     <param>
        <post_avblfldidx>703</post_avblfldidx>
        <shortname>GSD_MAX_LTG_THREAT2_ON_ENTIRE_ATMOS</shortname>
-       <pname>LTGTHREAT2</pname>
+       <pname>LTNGTH2</pname>
        <fixed_sfc1_type>entire_atmos</fixed_sfc1_type>
```

Comment:
Why change this one? It  is not related to this work.

Reply:
I will revert this.  SSEC / Wisconsin colleagues also want to have lightning threat output in RRFS, I guess they were going ahead and making changes in anticipation of that.